src/members/toker.json

```
@@ -4,13 +4,13 @@
   "status": "officer",
   "pronouns": "he/him/his",
   "class": "Class of 2026",
-  "socials": ["https://instagram.com/tenboker"],
-  "bio": "Possibly cool.",
+  "socials": ["btoker.dev"],
```

```suggestion
  "socials": ["https://btoker.dev"],
```

The `https://` has to be there for the link to work.